src/main/target/REVONANO/target.h

```diff
@@ -78,8 +78,11 @@
 #define USE_SPI_DEVICE_2
 
 #define USE_I2C
-#define USE_I2C_DEVICE_1        // UART1/FlexiPort (PB6,PB7)
```

Comment:
The comment looks like this is a broken out I2C port. This should probably stay enabled in addition to the one used for BARO.

Reply:
OK. Restored.